CHANGELOG.md

```diff
@@ -2,6 +2,91 @@
 
 ## Unreleased
 
+### API
+
+- Fix bug in `W3CBaggagePropagator` that caused `+` characters to be decoded as whitespace ` `.
+  [(#4898)](https://github.com/open-telemetry/opentelemetry-java/pull/4898)
+
+#### API Extensions
+
+* DEPRECATION: the `opentelemetry-extension-aws` module containing
+  various `AwsXrayPropagator` implementations has been deprecated for removal in next major version.
+  A copy of the code will instead be maintained
+  in [opentelemetry-java-contrib/aws-xray-propagator](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/aws-xray-propagator)
+  and published under
+  coordinates `io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:{version}`.
+  [(#4862)](https://github.com/open-telemetry/opentelemetry-java/pull/4862)
```

Comment:
I like the approach that `opentelemetry-java-instrumetnation` and `opentelemetry-specification` takes in including links to the PR corresponding to the change.